docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-panel-plugin-with-d3.md

```diff
@@ -25,22 +28,18 @@ In this tutorial, you'll:
 - Build a simple panel plugin to visualize a bar chart.
 - Learn how to use D3.js to build a panel using data-driven transformations.
 
-{{% class "prerequisite-section" %}}
-
 ### Prerequisites
 
 - Grafana 7.0
 - [LTS](https://nodejs.dev/en/about/releases/) version of Node.js
-- yarn
-  {{% /class %}}
 
 ## Set up your environment
 
-{{< docs/shared lookup="tutorials/set-up-environment.md" source="grafana" version="latest" >}}
+<SetupEnvironment />
 
 ## Create a new plugin
 
-{{< docs/shared lookup="tutorials/create-plugin.md" source="grafana" version="latest" >}}
+<CreatePlugin />
```

Comment:
CreatePlugin builds a datasource as things stand. We'd need to fix that into separate ones to use with a panel plugin